yesod-auth/ChangeLog.md

```diff
@@ -1,3 +1,7 @@
+## 1.6.5
+
+* Email: Immediately register with a password [#1389](https://github.com/yesodweb/yesod/issues/1389)
```

Comment:
Perhaps a few words on how to use the new functionality would be warranted here.